mpas_analysis/ocean/sst_timeseries.py

```diff
@@ -88,17 +107,22 @@ def sst_timeseries(config, streamMap=None, variableMap=None):
 
     if ref_casename_v0 != 'None':
         print '  Load in SST for ACMEv0 case...'
+        inRefDate = '0001-01-01 00:00:00'
+
         infiles_v0data = '{}/SST.{}.year*.nc'.format(indir_v0data,
                                                      ref_casename_v0)
```

Comment:
@xylar: for ACME v0 data shouldn't you also define outRefDate as year_offset+(0001-01-01)?